eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -68,28 +68,21 @@ Get-Content -Raw path/to/file.yml | CompatibleConvertFrom-Yaml
 #>
 function CompatibleConvertFrom-Yaml {
   param(
-    [Parameter(Mandatory=$true, ValueFromPipeline=$true)]
+    [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
     [string]$Content
   )
 
   if (!($Content)) {
     throw "Content to parse is a required input."
   }
 
-  # Initialize any variables or checks that need to be done once
-  $yqPresent = Get-Command 'yq' -ErrorAction SilentlyContinue
-  if (-not $yqPresent) {
+  if (-not $script:IsYamlInstalled) {
     . (Join-Path $PSScriptRoot PSModule-Helpers.ps1)
     Install-ModuleIfNotInstalled -WhatIf:$false "powershell-yaml" "0.4.7" | Import-Module
```

Comment:
@benbp should we add another helper called Import-ModuleIfNotAlready (or something like that) to make the helper handle the noop aspect of importing.

Reply:
Yeah I think that's a better approach. We could do some sort of convention under the hood for the script scoped variable. But I wouldn't hold up this change on it either.

Reply:
Ya'll mind if I take this `script` level variable over to `PSModule-Helpers.ps1` then? It's not like you're asking for a big change, just moving this change over there.

Reply:
Yeah that sounds good to me